lib/truffle/io/console.rb

```diff
@@ -45,6 +45,38 @@ def getpass(prompt = nil)
 
 
 class IO
+  def self.console(sym=nil, *args)
+    raise TypeError, "expected Symbol, got #{sym.class}" unless sym.nil? || sym.kind_of?(Symbol)
+
+    # klass = self == IO ? File : self
+    if defined?(@console) # using ivar instead of hidden const as in MRI
+      con = @console
```

Comment:
Minor: I'd rename `con` to console here for better readability.

Reply:
I just copied from JRuby, but I can rename.